public/app/features/inspector/utils/utils.ts

```diff
@@ -0,0 +1,37 @@
+import { AppEvents } from '@grafana/data';
```

Comment:
Maybe we can move those utils to their destination - dashboard-scene/inspect and reference in the old code if needed?

Reply:
@dprokop maybe, I think we can do this later the features/inspector is a bit messy now with shared code with old dashboard & explore, so leaving it here for now I think.